src/iasPET-load.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Method of loading the IAS Publisher Optimization tag if needed 
+ * see: gawkermedia/kinja-mantle/packages/standalone-ad-manager
+ */
+
+if (!window.__iasPET){
+    var useSSL = 'https:' === document.location.protocol;
+    var iasPET = document.createElement('script');
+
+    iasPET.async = true;
+    iasPET.type = 'text/javascript';
+    iasPET.src = (useSSL ? 'https:' : 'http:') + '//cdn.adsafeprotected.com/iasPET.1.js';
+
+    var node = document.getElementsByTagName('script')[1];
+// Inserting prior to second script because gpt needs to exist prior to any of the iasPET functionality being executed
+    node.parentNode.insertBefore(iasPET, node);
```

Comment:
The ordering probably doesn't mater, but `iasPET.js` needs to be on the page in some manner right? 

I feel like it's a better idea to have bulbs try to write it if it's not available, rather than just assuming people using bulbs in a standalone manner are perfect in self-loading that as a dependency on the page.